capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -132,7 +203,9 @@ function createContentString(name, items) {
 
                 </div>
                 <div id="posts" class="tab-pane fade">
-                    <h4>This is where the posts for ${name} will go.</h4>
+                    <h5>There are no posts for ${name} yet. Click
+                    <a href="../pages/comments.jsp">here</a>
+                    to be the first to make one.</h5>
```

Comment:
New line for ```<h5>``` and ```</h5>``` should be on their own lines I believe

Reply:
fixed! @andrewnguyen